#iffe determines wrongly _lib_qldexp and _lib_qfrexp
# both should be 0
--- lib/sfio/features/sfio.orig	2014-04-13 22:40:25.000000000 +0200
+++ lib/sfio/features/sfio	2014-09-08 13:49:38.410506603 +0200
@@ -11,8 +11,6 @@
 
 hdr string
 hdr math
-lib qfrexp
-lib qldexp
 
 hdr unistd
 hdr values
